test/parallel/test-openssl-ca-options.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+// This test checks the usage of --use-bundled-ca and --use-openssl-ca arguments
+// to verify that both are not used at the same time.
+const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+const assert = require('assert');
+const child_process = require('child_process');
+const result = child_process.spawnSync(process.execPath,
+                                      ['--use-bundled-ca', '--use-openssl-ca'],
```

Comment:
Can you line the start of this array and the start of the object on the following line up with the `p` in process.